get_session.go

```diff
@@ -0,0 +1,60 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package iotdb
+
+import (
+	"github.com/apache/iotdb-client-go/rpc"
+	"github.com/apache/thrift/lib/go/thrift"
+)
+
+const (
+	DefaultUser            = "root"
+	DefaultPasswd          = "root"
+	DefaultZoneId          = "Asia/Shanghai"
+	DefaultFetchSize int32 = 10000
+)
+
+//session config
+type Config struct {
```

Comment:
add `Validate` method for `Config` struct,
to check must be a valid `port` number, `username` and  `password` length, `fetchsize` cannot be a particularly large number, etc.